androidApp/src/main/java/kmm/rickandmorty/app/android/presentation/components/locations/LocationsScreen.kt

```diff
@@ -37,39 +34,42 @@ fun LocationsScreen() {
     when (state) {
         is Success -> {
             LazyColumn(
-                contentPadding = PaddingValues(top = 10.dp, bottom = 60.dp),
-                verticalArrangement = Arrangement.spacedBy(10.dp)
+                contentPadding = PaddingValues(top = dimensionResource(id = R.dimen.location_list_content_top_padding), bottom = dimensionResource(id = R.dimen.location_list_content_bottom_padding)),
+                verticalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.location_list_spacer))
             ) {
                 items((state as Success).data) { location ->
                     Location(location = location)
                 }
             }
         }
         is Error -> Unit
```

Comment:
Handle error case

Reply:
Will set up error components in a later pull request.

Reply:
Makes sense..